### PR TITLE
Add option to set the endpoint to fetch the jwks.json file

### LINKED ISSUE #18: Support to fetch the JWKS from different location than /.well-known/jwks.json
It is well-mentioned that jwksCache: the verifier will try to fetch the JWKS from the /.well-known/jwks.json endpoint each time it verifies a token. But this may not be always the case. This is the sitution for the company I am working on. Should such an assumption be so rigid? What about providing support for fetching the keys from other location? What about taking this param as a config param?

### PR DESCRIPTION
**Pull request by luisrudge**
fix #18

### CODE REVIEW

File: src/helpers/jwks.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ function getJWKS(options, cb) {
       var key;
 
       if (err) {
-        cb(err);
+        return cb(err);
```
Comment (joshcanhelp) [inline src/helpers/jwks.js]: Why did this change? Big behavior difference but I'm not seeing any other changes to how this is handled.

File: src/helpers/jwks.js [hunk 2]
```diff
@@ -34,7 +34,7 @@ function getJWKS(options, cb) {
         }
       }
 
-      cb(null, process(matchingKey));
+      return cb(null, process(matchingKey));
```
Comment (joshcanhelp) [inline src/helpers/jwks.js]: Same here

File: src/index.js [hunk 3]
```diff
@@ -14,6 +14,7 @@ var supportedAlgs = ['RS256'];
  * @param {String} parameters.audience identifies the recipients that the JWT is intended for
  * and should match the `aud` claim
  * @param {Object} [parameters.jwksCache] cache for JSON Web Token Keys. By default it has no cache
+ * @param {String} [parameters.jwksURI] Endpoint to fetch the jwks.json file
```
Comment (joshcanhelp) [inline src/index.js]: "URI to fetch the JSON Web Key Set (JWKS)" from the README

File: src/helpers/jwks.js (hunk 1 above)
Comment (luisrudge) [reply src/helpers/jwks.js]: it was actually wrong. see a similar PR here: https://github.com/auth0/idtoken-verifier/pull/14/files
if you don't return (or run the second part in an `else`), the code will actually keep running, which is not the desired effect.

**Pull request merged**